c/parallel/src/reduce.cu

```diff
@@ -206,13 +181,7 @@ struct __align__({2}) storage_t {{
 {3}
 {4}
 {5}
-using device_reduce_policy = {6}::MaxPolicy;
-
-#include <cub/detail/ptx-json/json.h>
-__device__ consteval auto& policy_generator() {{
-  return ptx_json::id<ptx_json::string("device_reduce_policy")>()
-    = cub::detail::reduce::ReducePolicyWrapper<device_reduce_policy::ActivePolicy>::EncodedPolicy();
-}};
```

Comment:
Maybe it'd be just a debug mode thing, but I'd like us to have some sort of a check here that the policy we've computed on the host is, in fact, the same policy that we computed in the JITed code. Right now that's by construction, but I worry slightly about running into weird esoteric bugs thanks to nvrtc behaving differently and/or there being some weird header mismatch somewhere, and then having the worst time trying to figure out what's going on when we could catch it here.